### DISCUSSION TITLE
`set-column-width` could grow beyond the monitor sizes

### DISCUSSION THREAD

**Discussion by kchibisov** [YaLTeR/niri#49] [General]
I don't think there's a value in such a thing given that the window can't be seen naturally anymore after that. Clamping to (combined)output resolution should desirable.

**Comment by YaLTeR** [comment]
There's actually a little value, but it will need window screenshots implemented. I used to make a Hotspot window larger than the monitor and window-screenshot it to get a flame graph with more of untruncated function names visible.

**Comment by kchibisov** [comment]
At this point you can just use svg and resize the graph to the width you want it to be.

**Comment by zefr0x** [comment]
I don't see any problem with this functionality if it was supported by the scrolling functionality.

I would like to be able to scroll over the same window when it is overflowing the monitor. The current behaviour is that it will jump to next window and truncate any overflowing part of the first window.
